### ISSUE TITLE
Chrome extension source code

### ISSUE THREAD

**Issue by cashflow1** [languagetool-org/languagetool#4342]
Hello!

I'm hoping to inspect the chrome extension source code. In the `language-broswer-addon` repo, it says the code has been rewritten under a new license: https://github.com/languagetool-org/languagetool-browser-addon/commit/457ba98e55286e77f74eed84e4b20fcdf863280d. Can anybody point me to where the new code lives?

Thanks!
Chris

**Comment by danielnaber** [comment]
The new code is not Open Source.

**Comment by digitalfu** [comment]
This is troubling, and I would like to understand the rationale behind the decision.
It seems a little underhanded to supplant an open-source add-on with a closed source one through the same release channel (especially if built using the web extension open standard and OSS libraries).

Can you elaborate or point us to the announcement that was made when the switch occurred?

**Comment by czadikem** [comment]
Why is this no longer open source?  It defeats the purpose of open source and makes me wonder what you guys have to hide.  Can you please enlighten me and everyone else why you just had to relicense this wonderful product and make it closed source?

**Comment by vuon9** [comment]
If this becomes a closed source, so there is nothing worth. Because the tool by itself is already not so useful (just cheap). I had to prioritize it as the highest choice over others just because it's OOS. Very disappointed!

**Comment by czadikem** [comment]
Yeah this is very annoying and there is no good alternative yet.  Languagetool was great but now it is starting to be a big data collector.

**Comment by igorsantos07** [comment]
Incredible how the member just said "not open source" and blatantly closed the issue.

**Comment by hamirmahal** [comment]
Has anyone come across any decent open-source Chrome extension alternatives?
